feat: addition of python-sdk example + refactor

parth-khiera opened the pull request.

parth-khiera committed:
feat: addition of python-sdk example + refactor

parth-khiera committed:
fix: removal of model list ref.

parth-khiera committed:
fix: rename readme

jainavana commented on non-sdk-examples/README.md:
direct-integration-example/README.md

jainavana commented on sdk-examples/README.md:
python-sdk-examples

jainavana reviewed (commented):
minor folder renaming. 
Also now README empty on main repo, we can maybe add something there later. Once folders renamed, push and merge without asking for review.

parth-khiera committed:
feat: add direct integration and python SDK examples

This commit introduces two new sets of examples:
1. Direct integration examples showing how to interact with the API via websockets and HTTP
2. Python SDK examples demonstrating local/remote/streaming transcription

The examples include complete implementations with error handling, logging, and documentation to help developers integrate with the Bodhi speech recognition platform.
